app/competition-form-types.ts

```diff
@@ -0,0 +1,10 @@
+export interface CompetitionFormInput {
```

Comment:
create and add this to a folder called 'types'. Or if this type is not used anywhere else other than the form include it there. In the mobile dir we use this type for creating a comp since the types are very similar to a normal competition 

`Partial<Competition> & { startDate: Date; endDate: Date }`